docs/how-tos/using-sentry-locally.md

```diff
@@ -10,31 +10,35 @@ FxA uses [sentry.io](https://sentry.io) to monitor errors and performance in pro
 
 To enable sentry for local development, simply set these environment variables in your root-level `.env` file (that you may need to create) to their respective DSN settings. When these variables are defined, sentry will be activated. 
 
-- SENTRY_ENV=local
-- [SENTRY_DSN_ADMIN_PANEL](https://sentry.io/settings/mozilla/projects/fxa-admin-panel/keys/)
-- [SENTRY_DSN_AUTH](https://sentry.io/settings/mozilla/projects/fxa-auth/keys/)
-- [SENTRY_DSN_CONTENT](https://sentry.io/settings/mozilla/projects/fxa-content/)
-- [SENTRY_DSN_CUSTOMS](https://sentry.io/settings/mozilla/projects/fxa-customs/)
-- [SENTRY_DSN_EVENT_BROKER](https://sentry.io/settings/mozilla/projects/fxa-event-broker/keys/)
-- [SENTRY_DSN_GRAPHQL_API](https://sentry.io/settings/mozilla/projects/fxa-graphql-api/keys/)
-- [SENTRY_DSN_PAYMENTS](https://sentry.io/settings/mozilla/projects/fxa-payments/keys/)
-- [SENTRY_DSN_PROFILE](https://sentry.io/settings/mozilla/projects/fxa-profile/keys/)
-- [SENTRY_DSN_SUPPPORT_PANEL](https://sentry.io/settings/mozilla/projects/fxa-support-panel/)
-- [SENTRY_DSN_BROWSERID](https://sentry.io/settings/mozilla/projects/fxa-browserid-verify/keys/)
+SENTRY_ENV=local
+[SENTRY_DSN_ADMIN_PANEL](https://sentry.io/settings/mozilla/projects/fxa-admin-panel/keys/)=?
+[SENTRY_DSN_AUTH](https://sentry.io/settings/mozilla/projects/fxa-auth/keys/)=?
+[SENTRY_DSN_CONTENT](https://sentry.io/settings/mozilla/projects/fxa-content/)=?
+[SENTRY_DSN_CUSTOMS](https://sentry.io/settings/mozilla/projects/fxa-customs/)=?
+[SENTRY_DSN_EVENT_BROKER](https://sentry.io/settings/mozilla/projects/fxa-event-broker/keys/)=?
+[SENTRY_DSN_GRAPHQL_API](https://sentry.io/settings/mozilla/projects/fxa-graphql-api/keys/)=?
+[SENTRY_DSN_PAYMENTS](https://sentry.io/settings/mozilla/projects/fxa-payments/keys/)=?
+[SENTRY_DSN_PROFILE](https://sentry.io/settings/mozilla/projects/fxa-profile/keys/)=?
+[SENTRY_DSN_SUPPPORT_PANEL](https://sentry.io/settings/mozilla/projects/fxa-support-panel/)=?
+[SENTRY_DSN_BROWSERID](https://sentry.io/settings/mozilla/projects/fxa-browserid-verify/keys/)=?
```

Comment:
@dschom What is the `=?` doin?

https://mozilla.github.io/ecosystem-platform/how-tos/using-sentry-locally looks a little 🐟y.

Reply:
I was trying to indicate they need to be set to something, and give a .env style format. Seeing it live, it doesn't really look quite right though.

Reply:
I've open a follow up [PR](https://github.com/mozilla/ecosystem-platform/pull/232) since this was already merged.